ansible/inventories/production/group_vars/inventory-next/vars.yml

```diff
@@ -35,12 +35,14 @@ inventory_app_repo_branch: inventory_ckan_2.8
 newrelic_app_name: inventory-next
 newrelic_enabled: true
 
-inventory_ckan_plugins_additional: [s3filestore]
+inventory_ckan_plugins_additional: [s3filestore saml2auth]
 
 # saml2 authentication
-inventory_ckan_saml2_enabled: false
+inventory_ckan_saml2_enabled: true
 ckan_production_ini_template: etc/ckan/production.ini
 inventory_ckan_who_ini_path: etc_ckan_who.default.ini.j2
```

Comment:
To revert previous [disabling commit](https://github.com/GSA/datagov-deploy/pull/2648/commits/d35763dfb526ed7fcf6439e42fe9d82a29ba279a), the `ckan_production_ini_template` and `inventory_ckan_who_ini_path` should also change back.

Reply:
Thanks! https://github.com/GSA/datagov-deploy/pull/2759